integration-cli/docker_utils.go

```diff
@@ -550,6 +550,10 @@ func pullImageIfNotExist(image string) (err error) {
 	return
 }
 
+func dockerCmdWithError(c *check.C, args ...string) (string, int, error) {
```

Comment:
Ah you're right, I did it so it would be almost the same signature as ``dockerCmd`` but it could go though :sweat_smile:.